---
title: "Benchmarking cell type annotation methods for 10x Xenium spatial transcriptomics data"
collection: publications
category: manuscripts
permalink: /publication/2025_Cheng_BMC_Bioinformatics
excerpt: 'This paper is about benchmarking five different reference-based cell type annotation methods for 10x Xenium data.'
date: 2025-01-20
venue: 'BMC Bioinformatics'
paperurl: 'https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-025-06044-0'
citation: 'Cheng, J., Jin, X., Smyth, G. K., and Chen, Y. (2025). &quot;Benchmarking cell type annotation methods for 10x Xenium spatial transcriptomics data.&quot; <i>BMC Bioinformatics</i>. 26(1), 22.'
---

We compared performance of five reference-based methods (*SingleR*, *Azimuth*, *RCTD*, *scPred* and *scmapCell*) with the marker-gene-based manual annotation method on an imaging-based Xenium data of human breast cancer. A practical workflow has been demonstrated for preparing a high-quality single-cell RNA reference, evaluating the accuracy, and estimating the running time for reference-based cell type annotation tools. *SingleR* was the best performing reference-based cell type annotation tool for the Xenium platform, being fast, accurate and easy to use, with results closely matching those of manual annotation.